src/test/javascript/spec/component/shared/updating-result.spec.ts

```diff
@@ -0,0 +1,155 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { LocalStorageService, SessionStorageService } from 'ngx-webstorage';
+import { MomentModule } from 'ngx-moment';
+import * as moment from 'moment';
+import { TranslateModule } from '@ngx-translate/core';
+import { AccountService, JhiLanguageHelper, WindowRef } from 'app/core';
+import { ChangeDetectorRef, DebugElement, SimpleChange, SimpleChanges } from '@angular/core';
+import { SinonStub, spy, stub } from 'sinon';
+import { BehaviorSubject, of } from 'rxjs';
+import * as chai from 'chai';
+import * as sinonChai from 'sinon-chai';
+import { AceEditorModule } from 'ng2-ace-editor';
+import { TreeviewModule } from 'ngx-treeview';
+import { CodeEditorFileService } from 'app/code-editor';
+import { ArTEMiSTestModule } from '../../test.module';
+import { MockParticipationWebsocketService, MockSyncStorage } from '../../mocks';
+import { Result, ResultComponent, UpdatingResultComponent } from 'app/entities/result';
+import { ArTEMiSSharedModule } from 'app/shared';
+import { Participation, ParticipationWebsocketService } from 'app/entities/participation';
+import { MockAccountService } from '../../mocks/mock-account.service';
+import { Exercise } from 'app/entities/exercise';
+
+chai.use(sinonChai);
+const expect = chai.expect;
+
+describe('UpdatingResultComponent', () => {
+    let comp: UpdatingResultComponent;
+    let fixture: ComponentFixture<UpdatingResultComponent>;
+    let debugElement: DebugElement;
+    let participationWebsocketService: ParticipationWebsocketService;
+
+    let subscribeForLatestResultOfParticipationStub: SinonStub;
+    let subscribeForLatestResultOfParticipationSubject: BehaviorSubject<Result>;
+
+    const exercise = { id: 20 } as Exercise;
+    const student = { id: 99 };
+    const gradedResult1 = { id: 10, rated: true, completionDate: moment('2019-06-06T22:15:29.203+02:00') } as Result;
+    const gradedResult2 = { id: 11, rated: true, completionDate: moment('2019-06-06T22:17:29.203+02:00') } as Result;
+    const ungradedResult1 = { id: 12, rated: false, completionDate: moment('2019-06-06T22:25:29.203+02:00') } as Result;
+    const ungradedResult2 = { id: 13, rated: false, completionDate: moment('2019-06-06T22:32:29.203+02:00') } as Result;
+    const results = [gradedResult2, ungradedResult1, gradedResult1, ungradedResult2] as Result[];
+    const initialParticipation = { id: 1, exercise, results, student } as Participation;
+    const newGradedResult = { id: 14, rated: true } as Result;
+    const newUngradedResult = { id: 15, rated: false } as Result;
+
+    beforeEach(async () => {
+        return TestBed.configureTestingModule({
+            imports: [TranslateModule.forRoot(), ArTEMiSTestModule, AceEditorModule, TreeviewModule.forRoot(), ArTEMiSSharedModule, MomentModule],
```

Comment:
i think we can remove `AceEditorModule` here

Reply:
good catch! this will make the tests run faster :)